contracts/src/v0.8/ccip/package.json

```diff
@@ -0,0 +1,92 @@
+  "version": "1.5.0",
+  "description": "Chainlink smart contracts for CCIP",
+  "author": "Chainlink devs",
+  "license": "BUSL-1.1",
+  "private": false,
+  "scripts": {
+    "test": "hardhat test --parallel",
+    "size": "hardhat size-contracts",
+    "clean": "hardhat clean",
+    "compile:native": "./../../../scripts/native_solc_compile_all",
+    "compile": "hardhat compile",
+    "coverage": "hardhat coverage",
+    "prepublishOnly": "pnpm compile && ./../../../scripts/prepublish_generate_abi_folder",
+    "publish-beta": "pnpm publish --tag beta",
+    "publish-prod": "pnpm publish --tag latest",
+    "solhint": "solhint --config ./../../../.solhint.json --max-warnings 0 \"./**/*.sol\"",
+    "solhint-test": "solhint --config \"./../../../.solhint-test.json\"  --ignore-path \"./../../../.solhintignore-test\" --max-warnings 0 \"./**/*.sol\""
+  },
+  "files": [
+    "LICENSE.md",
+    "*.sol",
+    "**/*.sol",
+    "abi/v0.8"
+  ],
+  "pnpm": {
+    "_comment": "See https://github.com/ethers-io/ethers.js/discussions/2849#discussioncomment-2696454",
+    "_comment2_logger": "See https://github.com/ethers-io/ethers.js/issues/379 we pin this version since that's what was used in the old yarn lockfile",
+    "overrides": {
+      "@ethersproject/logger": "5.0.6"
+    }
+  },
+  "engines": {
+    "node": ">=18",
+    "pnpm": ">=9"
+  },
+  "devDependencies": {
+    "@ethereum-waffle/mock-contract": "^3.4.4",
+    "@ethersproject/abi": "~5.7.0",
+    "@ethersproject/bignumber": "~5.7.0",
+    "@ethersproject/contracts": "~5.7.0",
+    "@ethersproject/providers": "~5.7.2",
+    "@nomicfoundation/hardhat-chai-matchers": "^1.0.6",
+    "@nomicfoundation/hardhat-ethers": "^3.0.8",
+    "@nomicfoundation/hardhat-network-helpers": "^1.0.12",
+    "@nomicfoundation/hardhat-verify": "^2.0.11",
+    "@typechain/ethers-v5": "^7.2.0",
+    "@typechain/hardhat": "^7.0.0",
+    "@types/cbor": "~5.0.1",
+    "@types/chai": "^4.3.20",
+    "@types/debug": "^4.1.12",
+    "@types/deep-equal-in-any-order": "^1.0.3",
+    "@types/mocha": "^10.0.8",
+    "@types/node": "^20.16.10",
+    "@typescript-eslint/eslint-plugin": "^7.18.0",
+    "@typescript-eslint/parser": "^7.18.0",
+    "abi-to-sol": "^0.6.6",
+    "cbor": "^5.2.0",
+    "chai": "^4.5.0",
+    "debug": "^4.3.7",
+    "deep-equal-in-any-order": "^2.0.6",
+    "eslint": "^8.57.1",
+    "eslint-config-prettier": "^9.1.0",
+    "eslint-plugin-prettier": "^5.2.1",
+    "ethers": "~5.7.2",
+    "hardhat": "~2.20.1",
+    "hardhat-abi-exporter": "^2.10.1",
+    "hardhat-ignore-warnings": "^0.2.6",
+    "moment": "^2.30.1",
+    "prettier": "^3.3.3",
+    "prettier-plugin-solidity": "^1.4.1",
+    "solhint": "^5.0.3",
+    "solhint-plugin-chainlink-solidity": "git+https://github.com/smartcontractkit/chainlink-solhint-rules.git#v1.2.1",
+    "solhint-plugin-prettier": "^0.1.0",
+    "ts-node": "^10.9.2",
+    "typechain": "^8.2.1",
+    "typescript": "^5.6.2"
+  },
+  "dependencies": {
```

Comment:
TODO: run depcheck or another tool to whittle these down to what's actually needed.